### PR TITLE
Logo now redirects to home page - #78

### PR DESCRIPTION
**Pull request by hur**
Did some work on issue #78, if this isn't the way you want it to be done, please tell me.

### CODE REVIEW

File: components/header.js [hunk 1]
```diff
@@ -73,7 +73,9 @@ export default props => {
         <div className="header__container">
           <nav>
             <div className="nav__logo">
-              <img src="/static/favicons/android-chrome-192x192.png" alt="" />
+              <a href="/">
```
Comment (vinaypuppal) [inline components/header.js]: Use nextjs Link[https://github.com/zeit/next.js/#with-link] component for this instead of regular anchor tag.

Comment (hur) [reply components/header.js]: Okay, will work on that.

**Comment by hur** [comment]
You seem to use item.path type structures for moth pats - I'm not familiar with that, should it be implemented here? Also, the link object wouldn't hover so I tried adding `cursor: pointer;` to the relevant css part. However, there seems to be a hidden object making the hovering functionality kind of wonky ([See here](https://i.imgur.com/tCyeW23.png), I think it has to do with the mobile navigation elements - when hovering over that part of the logo, the cursor will stay normal.

**Comment by vinaypuppal** [comment]
@hur Oh okay I just saw your last comment. Let me debug

**Comment by vinaypuppal** [comment]
@hur 
> You seem to use item.path type structures for moth pahts - I'm not familiar with that, should it be implemented here?

What you implemented is fine no need to use `item.path` here 

> Also, the link object wouldn't hover so I tried adding cursor: pointer;

Thanks

> However, there seems to be a hidden object making the hovering functionality kind of wonky, I think it has to do with the mobile navigation elements - when hovering over that part of the logo, the cursor will stay normal.

Yes you are correct. Its because of hidden checkbox which we use for mobile menu toggling.
https://github.com/coderplex/coderplex/blob/78a8cf164d78d9df30d14b1dca900670ea355300/components/header.js#L187-L189
we have hidden it using `opacity` property which is causing this issue. Can you make it `display: none` and show it only on mobile for this you can change back it to `display: block` over here in media queries https://github.com/coderplex/coderplex/blob/78a8cf164d78d9df30d14b1dca900670ea355300/components/header.js#L253-L262

**Comment by hur** [comment]
Nevermind the previous changes, I made a mistake, will fix now.

**Comment by hur** [comment]
That should do it (I think). Thanks for the explanation, I appreciate it - helps me learn.

**Comment by duttakapil** [comment]
Congratulations @hur :D

**Pull request merged**